libexec/scoop-search.ps1

```diff
@@ -106,17 +125,27 @@ Get-LocalBucket | ForEach-Object {
         Write-Host "'$name' bucket:"
         $res | ForEach-Object {
             $item = "    $($_.name) ($($_.version))"
-            if($_.bin) { $item += " --> includes '$($_.bin)'" }
+            if($_.binaries) { $item += " --> $($_.binaries -join ', ')" }
+            if($_.description) { $item += "`n        $($_.description)" }
             $item
```

Comment:
Do not just put variables into stream. Use write-host or return in whole codebase